SIDFactoryII/source/utils/configfile.h

```diff
@@ -19,6 +19,10 @@ namespace Utility
 	public:
 		ConfigFile(const Foundation::IPlatform& inPlatform, const std::string& inFilename, const std::vector<std::string>& inValidSectionTags);
 
+		ConfigFile(ConfigFile& inOther) = delete;
+		ConfigFile(ConfigFile&& inOther) = delete;
+		ConfigFile(const ConfigFile& inOther) = delete;
+
```

Comment:
Probably also need to delete the assingment and move operators. 

The non const copy comstructor you do not need to delete, as it's not a default constructor.

Reply:
Deleted https://en.cppreference.com/w/cpp/language/move_assignment operator
Removed the line `ConfigFile(const ConfigFile& inOther) = delete;`